docs/gettingStarted.md

```diff
@@ -28,28 +28,39 @@ There are two game modes available, `'custom'` and `'random'`.
 To use a defined word, for example a word in your language, you can use the `option.word` option.
 
 ```js
-await hangman.create(message.channel, 'random', { word: 'discord' })
+await hangman.create(interaction, 'random', { word: 'discord' })
 ```
 The word provided must not contain spaces, numbers or special characters. Only uppercase, lowercase and accented letters or not.
 If the game mode is set to `'custom'`, this option will be ignored.
 
 ## Players option
 
 You can automatically add certain players to a game and start the game by using the `option.players` option.
-One particular use of this option, is to automatically start a game with the players value being the message author which creates a quick singleplayer game.
+One particular use of this option, is to automatically start a game with the players value being the interaction user which creates a quick singleplayer game.
 
 ```js
-hangman.create(message.channel, 'random', { players: [message.author] })
+hangman.create(interaction, 'random', { players: [interaction.user] })
 ```
 The players value provided must be an array of discord users.
 
+## Language option
```

Comment:
I'm not really in favor of adding a language system in the module. I think it's better to provide a small list of default words so that the module is self-sufficient, but let the user have the possibility to customize the available words himself. If we add different languages, we will have to add more and more as the requests come in. And the translation with Google translation (or Deepl I don't know haha) is not that great... @Spongecade what do you think? (if you have time to answer)

Reply:
Resolved